### PR TITLE
Image Gallery snippet

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**
Image Gallery snippet for website with hover effect.

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (elicoidal) [inline oca_dependencies.txt]: @JayVora-SerpentCS Thanks for your contribution to the OCA. 
Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

Please remove this file

File: website_snippet_image_gallery/README.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Website Snippet Image Gallery
+==================================
```
Comment (elicoidal) [inline website_snippet_image_gallery/README.rst]: until the end of the sentence

File: website_snippet_image_gallery/README.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Description
+-----------
+
+This module was written to add the functionality of image gallery in website with hover effects.
```
Comment (elicoidal) [inline website_snippet_image_gallery/README.rst]: This module adds the functionality of image gallery in website with hover effects.

File: website_snippet_image_gallery/README.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+Usage
+------------
```
Comment (elicoidal) [inline website_snippet_image_gallery/README.rst]: double until en of sentence

File: website_snippet_image_gallery/README.rst [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+To use this module, you need to:
+
+To use this module you just need to drag and drop image-gallery snippet in your page.
+For further information, please visit:
+
+https://www.odoo.com/forum/help-1
+
+Credits
```
Comment (elicoidal) [inline website_snippet_image_gallery/README.rst:23]: Miss the bug tracker section

File: website_snippet_image_gallery/README.rst [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+* Jay Vora <jay.vora@serpentcs.com>
+* Meet Dholakia <m.dholakia.serpentcs@gmail.com>
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
```
Comment (elicoidal) [inline website_snippet_image_gallery/README.rst]: https

File: website_snippet_image_gallery/README.rst [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline website_snippet_image_gallery/README.rst:56]: https

File: website_snippet_image_gallery/__init__.py [hunk 8]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline website_snippet_image_gallery/__init__.py:1]: Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: website_snippet_image_gallery/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+##############################################################################
+
+{
+    'name': 'Website Snippet Image Gallery',
+    'category': 'Website',
+    'author': 'Serpent Consulting Services Pvt. Ltd.',
+    'website': 'http://www.serpentcs.com',
+    'summary': 'Adds image gallery in website.',
+    'version': '8.0.1.0.0',
+    'description': """
```
Comment (elicoidal) [inline website_snippet_image_gallery/__openerp__.py]: remove description

**Comment by elicoidal** [comment]
@JayVora-SerpentCS 

What is the licensing of the provided photos? Please make sure they are creative common and add in the README file the information

**Comment by rafaelbn** [comment]
Please @JayVora-SerpentCS could you please review this PR? Comments from @elicoidal and Travis
Thanks

**Comment by simahawk** [comment]
@JayVora-SerpentCS still working on this?
Closing as is 2yrs old and no answer.
Feel free to re-open if needed, thanks!

**Comment by JayVora-SerpentCS** [comment]
yes close! :(

**Pull request closed without merging**